src/main/fc/tasks.c

```diff
@@ -539,6 +544,10 @@ void tasksInit(void)
     setTaskEnabled(TASK_ALTHOLD, true);
 #endif
 
+#ifdef USE_POS_HOLD_MODE
+    setTaskEnabled(TASK_POSHOLD, true);
```

Comment:
```suggestion
    setTaskEnabled(TASK_POSHOLD, sensors(SENSOR_GPS);
```

Reply:
thanks

Reply:
edit - that change caused GPS to fail; needs to be that the feature is enabled, because I don't think we know if the GPS hardware using sensors() at the time we assign tasks.  Used the feature method and is OK.